### ISSUE TITLE
onMessageReceived not called, when app is in killed state [Android]

### ISSUE THREAD

**Issue by sumesh1993** [evollu/react-native-fcm#504]
Hi @evollu 

I want when the app is killed(removed from the task manager) and a notification is received. A function is called depending upon the data inside that notification.
How can i achieve this ?

onMessageReceived is called when the app is in foreground or background but not when its completely killed.

"react-native": "0.47.1",
"react-native-fcm": "^8.0.0",
testing on moto g4 play (android 6.0.1);

i am looking for something like the onNotification() method in this [repo](https://github.com/zo0r/react-native-push-notification).
Basically i am looking for a function which is called when a remote or local notification is opened or received

**Comment by evollu** [comment]
did you send payload with only `data` property and no `notification` property? Otherwise it won't trigger `onMessageReceived`. It is a FCM behavior
